### PR TITLE
cloud: update cluster creation steps

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: develop/dev-guide-build-cluster-in-cloud.md [hunk 1]
```diff
@@ -34,9 +34,9 @@ This document walks you through the quickest way to get started with TiDB Cloud.
     To create a new Serverless Tier cluster on your own, take the following operations:
 
     1. Click **Create Cluster**.
-    2. On the **Create Cluster** page, **Serverless Tier** is selected by default. Update the default cluster name if necessary, select a target region of your cluster, and then click **Create**. Your Serverless Tier cluster will be created in approximately 30 seconds.
+    2. On the **Create Cluster** page, **Serverless** is selected by default. Select a target region of your cluster, update the default cluster name if necessary, and then click **Create**. Your Serverless Tier cluster will be created in approximately 30 seconds.
 
-4. Click the target cluster name to go to its overview page, and then click **Connect** in the upper-right corner. A connection dialog box is displayed.
+4. On the cluster overview page, and then click **Connect** in the upper-right corner. A connection dialog box is displayed.
```
Comment (qiancai) [inline develop/dev-guide-build-cluster-in-cloud.md:39]:
```suggestion
4. On the cluster overview page, click **Connect** in the upper-right corner. A connection dialog box is displayed.
```

File: tidb-cloud/create-tidb-cluster.md [hunk 2]
```diff
@@ -100,36 +104,38 @@ To create a Dedicated Tier cluster, take the following steps:
...
+    3. Update the default cluster name and port number if necessary.
+    4. If this is the first cluster of your current project and CIDR has not been configured for this project, you need to set the project CIDR. If you do not see the **Project CIDR** field, it means that CIDR has already been configured for this project.
 
-5. Configure the [cluster size](/tidb-cloud/size-your-cluster.md) for TiDB, TiKV, and TiFlash (optional) respectively, and then click **Next**.
+        > **Note:**
+        >
+        > When setting the project CIDR, avoid any conflicts with the CIDR of the VPC where your application is located. You cannot modify your project CIDR once it is set.
 
-6. Confirm the cluster information on the page and the billing information in the lower-left corner.
+4. Confirm the cluster and billing information on the right.
```
Comment (ran-huang) [inline tidb-cloud/create-tidb-cluster.md:124]:
```suggestion
4. Confirm the cluster and billing information on the right side.
```

**Review by ran-huang — APPROVED** [review]
rest LGTM

**Review by shizn — APPROVED** [review]

**Pull request merged**